test/test_blob.rb

```diff
@@ -272,6 +272,8 @@ def test_language
           assert blob.language.nil?, "A language was found for #{filepath}"
         elsif language == 'Generated'
           assert blob.generated?, "#{filepath} is not a generated file"
+        elsif language == 'Solidity' && filename.match(/ignored\d\.sol$/)
+          assert !blob.language, "#{filepath} should not match a language"
```

Comment:
I'm assuming these special cases won't be needed once these files are moved out of the Solidity directory.

Reply:
[See above](https://github.com/github/linguist/pull/4936#discussion_r462903775). These files aren't in the `samples/Solidity` directory, only in the `test/fixtures/Solidity` directory (and if that's not a confusing enough distinction for you, our [`test_heuristics.rb`](https://github.com/github/linguist/blob/96ad1185828f44bb9b774328a584551ee57ed264/test/test_heuristics.rb#L20) file refers to the actual "sample" files as "fixtures"...